backend/api/api.csproj

```diff
@@ -30,7 +30,9 @@
     <PackageReference Include="Swashbuckle.AspNetCore" Version="6.6.2" />
     <PackageReference Include="MQTTnet" Version="4.3.6.1152" />
     <PackageReference Include="MQTTnet.Extensions.ManagedClient" Version="4.3.6.1152" />
-    <PackageReference Include="System.Linq.Dynamic.Core" Version="1.5.1" />
+    <PackageReference Include="System.Linq.Dynamic.Core" Version="1.5.1" >
+        <NoWarn>NU1903</NoWarn>
+    </PackageReference>

```

Comment:
Remove this and merge it even if the pipeline fails

Reply:
There was a test failing with this in. Good to have it temporarily so we can know that there are no tests failing, then remove the ignore so that the pipeline fails again

Reply:
So the test was failing but was not showing because this error stopped the pipeline before the test could fail?